recipes/watchdog-wms/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "watchdog-wms" %}
+{% set version = "1.2.0b" %}
+
+about:
+    home: https://www.bio.ifi.lmu.de/watchdog https://github.com/klugem/watchdog
+    license: GNU General Public License, Version 3
+    summary: 
+       Watchdog, a WMS for the automated and distributed analysis of large-scale experimental data. 
+       The software is implemented in Java and is thus platform-independent. 
+       Main feature include
+        - straightforward processing of replicate data
+        - support for distributed computer systems
+        - customizable error detection
+        - manual intervention into workflow execution
+        - GUI for workflow construction using pre-defined modules
+        - a helper script for creating new module definitions
+        - no restriction to specific programming languages
+        - provides a flexible plugin system for extending without modifying the original sources
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+build:
+  number: 0
+
+source:
+    fn: {{ name }}-{{ version }}.tar.gz
+    url: https://github.com/klugem/watchdog/archive/v{{ version }}.tar.gz
+    sha256: bda0751c47587036149bf838de9dd9b6d20a24f9192d022cfc5eebe66c1fc537
+
+requirements:
+  build:
+  run:
+      - openjdk >=8
+
+extra:
+  maintainers:
+   - Michael Kluge <michael.kluge@bio.ifi.lmu.de>
+  notes: |
+     Graphical user interface works only with javafx, which is currently not part of openjdk8.
+     Dependency should be changed to java-jdk when working again.
```

Comment:
Even though there is an older `java-jdk` package available, that is OpenJDK as well, IIRC.
So we'd actually need someone to create a new package for [OpenJFX](http://wiki.openjdk.java.net/display/OpenJFX/Main) over at `conda-forge`.

Reply:
Yes, I had that included in my local version but automated build failed caused by java-jdk dependency.